doc/src/build/zk_login.md

```diff
@@ -477,23 +477,31 @@ zkLogin can support providers that work with OpenID Connect built on top of the
 
 Traditional private key wallets demand users to consistently recall mnemonics and passphrases, necessitating secure storage to prevent fund loss from private key compromise.
 
-On the other hand, a zkLogin wallet only requires an ephemeral private key storage with session expiry and the OAuth login flow with expiry. A loss of ephemeral keys alone does not mean a loss of funds.
+On the other hand, a zkLogin wallet only requires an ephemeral private key storage with session expiry and the OAuth login flow with expiry. Forgetting ephemeral keys alone does not result in loss of funds, because a user can always sign in again to generate a new ephemeral key and a new ZK proof.
```

Comment:
```suggestion
On the other hand, a zkLogin wallet only requires an ephemeral private key storage with session expiry and the OAuth login flow with expiry. Forgetting ephemeral keys does not result in loss of funds, because a user can always sign in again to generate a new ephemeral key and a new ZK proof.
```